tests/forms_tests/tests/test_widgets.py

```diff
@@ -1997,3 +1997,19 @@ def test_form_label_association(self):
         # label tag is correctly associated with first rendered dropdown
         a = GetDate({'mydate_month': '1', 'mydate_day': '1', 'mydate_year': '2010'})
         self.assertIn('<label for="id_mydate_day">', a.as_p())
+
+
+class SelectWidgetTests(SimpleTestCase):
+
+    def test_deepcopy(self):
+        """
+        Ensure that __deepcopy__ copies all fields properly.
+        Refs #25085
+        """
+        widget = Select()
+        obj = copy.deepcopy(widget)
+        self.assertTrue(widget is not obj)
+        self.assertTrue(widget.choices == obj.choices)
```

Comment:
I wanted to make the difference between the two comparisons clear. Do you think it should be changed?